lldb/source/Plugins/TypeSystem/Swift/SwiftASTContext.cpp

```diff
@@ -3881,7 +3925,8 @@ void SwiftASTContext::ValidateSectionModules(
 
     // We have to increment the completion value even if we can't get the module
     // object to stay in-sync with the total progress reporting.
-    progress.Increment(++completion);
+    progress.Increment(++completion,
+                       module.GetFileSpec().GetFilename().AsCString());
```

Comment:
This looks wrong, you need to re-use the `module_name` from the loop range declaration.

```suggestion
    progress.Increment(++completion, module_name);
```

Reply:
Ah, I reused the original implementation grabbed the module name from its FileSpec so that's how I missed this